src/docs/integrations/slack/index.mdx

```diff
@@ -59,7 +56,23 @@ Click __Add a new Redirect URL__, enter the URL, and click __Add__. Set the URL
 
 Click __Save URLs__.
 
-On the same page under __Scopes__ in the __Select Permission Scopes__ dropdown menu, add:
+On the same page under __Scopes__ you'll need to add __Bot Scopes__ and __User Scopes__.
+
+Add the following scopes to __Bot Scopes__:
```

Comment:
It would be great to have a pre-made URL that adds these to the form. GitHub has this and we used it for [GitHub SSO docs](https://develop.sentry.dev/self-hosted/sso/#github-auth).